pkgs/development/tools/misc/trv/default.nix

```diff
@@ -0,0 +1,36 @@
+{stdenv, fetchurl, ocaml, findlib, camlp4, core, async, async_unix, re2,
+  async_extra, sexplib, async_shell, core_extended, async_find, cohttp, uri, tzdata}:
+
+let
+  ocaml_version = (builtins.parseDrvName ocaml.name).version;
+  version = "0.1.3";
+in
+
+assert stdenv.lib.versionOlder "4.02" ocaml_version;
+
+stdenv.mkDerivation {
+  name = "trv-${version}";
+
+  src = fetchurl {
```

Comment:
@pSub I maintain a number of ocaml packages. Should I be converting them all over to fetchFromGitHub?